annotation-processing/autoload/src/main/java/net/flintmc/processing/autoload/DetectableAnnotationProcessor.java

```diff
@@ -50,6 +50,7 @@
   private static final String MINECRAFT_VERSION_OPTION = "net.flintmc.minecraft.version";
 
   private static final String METAPROGRAMMING_PACKAGE = "net.flintmc.metaprogramming";
+  private static final String ELEMENT_KIND_CLASS = METAPROGRAMMING_PACKAGE + ".ElementKind";
```

Comment:
For all other classes in here the metaprogramming package is not prepended directly. Is there a reason this is done for this specific instance? See the `metaprogrammingClass` method